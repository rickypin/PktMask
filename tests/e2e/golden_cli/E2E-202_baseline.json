{
  "test_id": "E2E-202",
  "name": "Single VLAN Encapsulation",
  "input_file": "tests/data/tls/tls_1_2_single_vlan.pcap",
  "input_hash": "12eb83587b8a4d281ae858a376ab8c34dedca9d25c762e2b97dbf0747cba6260",
  "cli_args": [
    "--anon",
    "--mask"
  ],
  "output_hash": "4a59196a872edc8a92ce24bce9031e7598f14d414287d98584e1e94edb51ea02",
  "timestamp": "2025-10-09T21:31:39.139855",
  "version": "1.0-cli",
  "note": "Generated using CLI only - completely decoupled from API"
}