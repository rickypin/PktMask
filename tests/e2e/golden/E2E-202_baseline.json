{
  "test_id": "E2E-202",
  "name": "Single VLAN",
  "input_file": "tests/data/tls/tls_1_2_single_vlan.pcap",
  "input_hash": "12eb83587b8a4d281ae858a376ab8c34dedca9d25c762e2b97dbf0747cba6260",
  "config": {
    "dedup": false,
    "anon": true,
    "mask": true
  },
  "output_hash": "4a59196a872edc8a92ce24bce9031e7598f14d414287d98584e1e94edb51ea02",
  "output_file_size": 1571129,
  "packet_count": 2111,
  "stats": {
    "packets_processed": 2111,
    "packets_modified": 3379,
    "duration_ms": 8630.096197128296,
    "stages": [
      {
        "name": "AnonymizationStage",
        "packets_processed": 2111,
        "packets_modified": 2111,
        "duration_ms": 826.265811920166
      },
      {
        "name": "Mask Payloads",
        "packets_processed": 2111,
        "packets_modified": 1268,
        "duration_ms": 927.2129535675049
      }
    ]
  },
  "timestamp": "2025-10-09T19:40:10.607195",
  "version": "1.0"
}